src/js/components/Button/Button.js

```diff
@@ -92,10 +92,16 @@ const getIconColor = (paths = [], theme, colorProp, kind) => {
   return result[1] || undefined;
 };
 
+const tipString = tip => {
+  if (typeof tip === 'string') {
+    return tip;
+  }
+  return '';
```

Comment:
I think it will be cleaner to use `undefined` instead of an empty string.
With the current solution, we have an empty string of `aria-label` that is leaking to the DOM unnecessarily. This change may clean up the snapshots. 
Let me know your thoughts on this.

Reply:
Hi @ShimiSun, good call! I had originally tried to return `false`, but this didn't produce the expected result. That certainly did clean things up.

The only "ButtonKind" reference I could find in the codebase was StyledButtonKind, so presumably this is what you're referring to? I added the same assignment there.

Please let me know if you need anything else to make this work. Thanks!

Reply:
THANK YOU! It really cleaned up the PR, the number of changed files is so much more reasonable now :)

Yea, StyledButtonKind sounds like the right place, I don't see any changes there though, what am I missing? 

Also, one last request, we try to keep PRs as lean as possible to their content, can you please remove all the changes of empty spaces removals that are affecting the snapshots/README unnecessary changes? It will remove the clutter from the PR.

Thanks again for working on this.

Reply:
> THANK YOU! It really cleaned up the PR, the number of changed files is so much more reasonable now :)

No problem!

> Yea, StyledButtonKind sounds like the right place, I don't see any changes there though, what am I missing?

Line 276 in the same file? That lives inside of `<StyledButtonKind />`
 
> Also, one last request, we try to keep PRs as lean as possible to their content, can you please remove all the changes of empty spaces removals that are affecting the snapshots/README unnecessary changes? It will remove the clutter from the PR.

I can certainly do this. However, I should point out that these files (and changes) were generated by running the scripts in the [Grommet contribution guidelines](https://github.com/grommet/grommet/blob/master/CONTRIBUTING.md), not by any change I introduced from my side. Let me see if I can make this work for you.

> Thanks again for working on this.

No problem. I'm glad I'm able to help.

Reply:
Hey @ShimiSun, I believe all the line-break issues have been cleaned up.